### PR TITLE
Adds SNS Basic Permissions

### PR DESCRIPTION
**Pull request by plandino**
- Allows to use SNS Push Notifications
- Does not allow to create, modify or destroy Platform applications

### CODE REVIEW

File: infraestructure/aws/docs/sns-basic-permissions.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# SNS Basic Permissions
+
```
Comment (mdesanti) [inline infraestructure/aws/docs/sns-basic-permissions.md]: I would add a subtitle here that clearly explains that this permissions are for sending push notifications. Something like `### Sending push notifications`

Comment (plandino) [inline infraestructure/aws/docs/sns-basic-permissions.md]: Done! And changed a little bit the description.

**Comment by mdesanti** [comment]
@mishuagopian @epintos any feedback you would want to drop here?

File: infraestructure/aws/docs/sns-basic-permissions.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        "sns:CreatePlatformEndpoint",
+        "sns:DeleteEndpoint",
+        "sns:GetEndpointAttributes",
+        "sns:GetPlatformApplicationAttributes",
+        "sns:ListEndpointsByPlatformApplication",
+        "sns:Publish",
+        "sns:SetEndpointAttributes",
+      ],
+      "Resource": [
+        "arn:aws:sns:SNS-REGION:SNS-ARN
```
Comment (epintos) [inline infraestructure/aws/docs/sns-basic-permissions.md]: Yo could explain a little bit about this two parameters

**Comment by mishuagopian** [comment]
IMO, it's ok!

File: infraestructure/aws/docs/sns-basic-permissions.md (hunk 2 above)
Comment (plandino) [inline infraestructure/aws/docs/sns-basic-permissions.md]: @epintos done! It's ok or there is too much information?

**Pull request merged**